README.md

```diff
@@ -1,8 +1,45 @@
-# Jarvis Data Engineering Training
-1. [Linux cluster monitoring agent (Linux and SQL)](./linux_sql) In-progress
-2. [Core Java Apps](./core_java) In-progress
-3. [Springboot Trading REST API](./springboot) In-progress
-4. [Hadoop/Hive](./hadoop) In-progress
-5. [Spark/Scala](./spark) In-progress
-6. [Cloud/DevOps](./cloud_devops) In-progress
+# Hunain Khalid . Jarvis Consulting
 
+I am a recent graduate from York University's Computer Science program. I specialize in software development, from compiler-level code to machine learning. Although proficient, I continue to improve my skills in Docker, Java, SQL, Python, React.js, Git, Linux and more. As a full-stack software developer, I solve problems quickly, with optimized run time and extensive documentation. A course on computer security helped with enterprise-level solutions to computer security and security at the programming level. I am also an excellent communicator, team member and leader. Currently employed at Jarvis, I improve my skills through difficult projects and team collaboration in an agile environment.
```

Comment:
I am a recent graduate from York University's Computer Science program. I specialize in software development, including full-stack development, compiler-level coding, and machine learning. Although proficient, I continue to improve my skills in Docker, Java, SQL, Python, React.js, Git, Linux and more. As a full-stack software developer, I solve problems quickly, with optimized run time and extensive documentation. A course on computer security helped with enterprise-level solutions to computer security and security at the programming level. I am also an excellent communicator, team member and leader. Currently employed at Jarvis, I improve my skills through challenging projects and team collaboration in an agile environment.